# world_T_cam pose 
float32[3] cam_rot
float32[3] cam_trans

Tag[] tags
